packages/@glimmer/runtime/lib/component/curried-component.ts

```diff
@@ -1,63 +1,86 @@
-import { CapturedArguments, ComponentDefinition, Dict, Maybe } from '@glimmer/interfaces';
-import { Option, symbol } from '@glimmer/util';
+import { CapturedArguments, ComponentDefinition } from '@glimmer/interfaces';
+import { Option, symbol, assert, _WeakSet } from '@glimmer/util';
 import { VMArgumentsImpl } from '../vm/arguments';
+import { capabilityFlagsFrom } from '../capabilities';
+import { ComponentInstance } from '../compiled/opcodes/component';
 
-const CURRIED_COMPONENT_DEFINITION_BRAND: unique symbol = symbol('CURRIED COMPONENT DEFINITION');
+const INNER: unique symbol = symbol('INNER');
+const ARGS: unique symbol = symbol('ARGS');
+
+const CURRIED_COMPONENT_DEFINITIONS = new _WeakSet();
```

Comment:
This will break some of the shenanigans that I'm doing over in:

https://github.com/rwjblue/ember-template-invocation-location/blob/4d13448d972b069bdcf4c21293b6a86af618a881/addon/helpers/-template-invocation-info-is-component-or-helper.js#L6-L7

Can you double check / confirm that `isCurriedComponentDefinition` is exported from `@glimmer/runtime` itself?